src/main/java/com/dao/MakePost.java

```diff
@@ -8,7 +8,7 @@ public static void createPost()
 	}
 	public static void editPost()
 	{
-		System.out.println("Edit Post Successfully");
+		System.out.println("Edited Post Successfully");
 	}
 	public static void readPost()

```

Comment:
Please Change Method Name to "readingPost()"

Reply:
Also create deletePost and implement it